network-api/networkapi/campaign/views.py

```diff
@@ -58,6 +72,64 @@ def petition_submission_view(request, pk):
         }
     })
 
+    return send_to_sqs(gs_queue_url, message)
+
+
+# handle Salesforce petition data
+def petition_submission(request, petition):
+    cid = petition.campaign_id
+
+    if cid is None or cid == '':
+        return Response(
+            {'error': 'Server is missing campaign for petition'},
+            status=status.HTTP_500_INTERNAL_SERVER_ERROR,
+        )
+
+    data = {
+        "campaign_id": cid,
+        "first_name": request.data['givenNames'],
+        "last_name": request.data['surname'],
+        "email": request.data['email'],
+        # We already submit an email subscription separately
```

Comment:
@Pomax in a meeting today it was suggested that I'd need to have basket send a subscription when it saw this key come in as `True` (I assume it's boolean?), but this comment suggest that this would be redundant. Do you know if I should or should not fire off a subscription for the mozilla newsletter for the email address of the signer when this flag is true?

Reply:
I also thought this was going to/should happen Basket-side, but seem to recall Pomax had a reason for it happening on the Foundation site server side.

Reply:
We've had this in place effectively "since forever" so when that was brought up during the first or second meeting we decided it wasn't worth changing that. However, we're not married to doing things client-side in the slightest so if the PM decision is to do this in on the basket side we can remove this code pretty much whenever is convenient